source/includes/fact-connection-strings-onprem.rst

```diff
@@ -0,0 +1,27 @@
+:ref:`mongosync <c2c-mongosync>` uses a :ref:`MongoDB URI
+connection string <mongodb-uri>` to connect clusters:
+
+- The SRV connection scheme has the form:
+
+  .. code-block:: none
+
+     mongodb+srv://[username:password@][host.domain.TLD][/defaultauthdb][?options]
+
+- The standard URI connection scheme has the form:
+
+  .. code-block:: none
+  
+     mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]
+
+Specify the hostnames of the :binary:`mongod` instances the same way
+that they are listed in your replica set configuration.
+
+For :ref:`sharded clusters <sharding-sharded-cluster>`, specify the
+hostnames of the :binary:`mongos` instances instead of the
+:binary:`mongod` instances.
```

Comment:
[Suggestion]: I'd recommend 
```
:program:`mongod`
:program:`mongos`
``` 
I'm not sure that it matters under the hood, but it seems to be what we're transitioning to.